packages/plugins/privacy/src/NinetailedPrivacyPlugin.ts

```diff
@@ -108,13 +128,11 @@ export class NinetailedPrivacyPlugin extends NinetailedPlugin {
     return this._instance;
   }
 
-  private consent(accepted: boolean) {
+  private async consent(accepted: boolean) {
```

Comment:
Since this function doesn't return or await a promise, there's no need to mark it as async